doc/src/build/authorities.md

```diff
@@ -1,33 +1,33 @@
 ---
```

Comment:
Probably want to rename this file to `validators.md`

Reply:
Voluntarily, I've left that to a further PR along with the code changes. This makes review easier:
https://github.com/MystenLabs/sui/pull/1398/files#diff-4bf78b90fcc082845477a30625a3a7d864cac80492da8e1621ba853ff5e19a30R11

Reply:
Gotcha!